nodes/src/nodes/llamaparse/IInstance.py

```
@@ -329,12 +319,7 @@ def _process_document(self, document_data: bytes, mime_type: str):
             debug(f'LlamaParse Instance: Full traceback: {traceback.format_exc()}')
 
             # Create error message as markdown
-            error_text = (
-                f'# LlamaParse Processing Error\n\n'
-                f'**Error Type:** {type(e).__name__}\n**Error Message:** {str(e)}\n\n'
-                f'**File Type:** {mime_type}\n**File Size:** {len(document_data)} bytes\n\n'
-                f'*The document could not be processed due to an error in the LlamaParse service.*'
-            )
+            error_text = f'# LlamaParse Processing Error\n\n**Error Type:** {type(e).__name__}\n**Error Message:** {str(e)}\n\n**File Type:** {mime_type}\n**File Size:** {len(document_data)} bytes\n\n*The document could not be processed due to an error in the LlamaParse service.*'
```

Ruff (0.15.7)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)